packages/grid_client/src/modules/tfchain.ts

```diff
@@ -477,7 +477,7 @@ class TFChain implements blockchainInterface {
       });
     }
     await (
-      await client.termsAndConditions.accept({ documentLink: "https://library.threefold.me/info/legal/#/" })
+      await client.termsAndConditions.accept({ documentLink: "https://manual.grid.tf/knowledge_base/legal/terms_conditions_all3.html" })
```

Comment:
Are we sure that we won't be having a legal wiki?

Reply:
From what I've been told, KDS wants to remove library.threefold.me, so I suggested that we link to the manual's legal section, which is a copy/paste (with minor tweaks) of the library's legal section.

I hope that answers the question. LMK. thanks.